lightpath/ui/gui.py

```diff
@@ -196,6 +200,11 @@ def change_path_display(self, value=None):
         """
         Change the display devices based on the state of the control buttons
         """
+        self.loading_splash.move(self.geometry().center()
+                                 - self.loading_splash.rect().center())
```

Comment:
These coordinates aren't correct for me on first load, the initial splash appears awkwardly at the top of one of my monitors in a half-loaded/frozen state

Reply:
It either doesn't appear at all (most common) or appears rendering oddly at the top of where the screen eventually appears. I tried to get a screenshot but it's rare and I gave up trying to repeatedly open it.